packages/cli/package.json

```diff
@@ -32,6 +32,7 @@
     "d3-scale": "^3.0.0",
     "fs-extra": "^7.0.1",
     "inquirer": "^6.2.2",
+    "jStat": "^1.7.1",
```

Comment:
https://www.npmjs.com/package/jStat
"NOTICE: From now on use the package name jstat as jStat will no longer be maintained"